packages/app/galaxy/eggs

```diff
@@ -0,0 +1 @@
+../../../lib/galaxy/eggs
```

Comment:
Do we really need this? It exists only for (probably unnecessary) backward compatibility.